### DISCUSSION TITLE
11/16/22 Metaverse Standards Forum: Oversight

### DISCUSSION THREAD

**Discussion by funksoup** [omigroup/MSF-Delegates#35] [Oversight]
Metaverse Standards Forum: Oversight
Wednesday, 16 November 2022 from 10:00 AM-11:00 AM EST (15:00-16:00 UTC)

Agenda:
- Forum General Update
- Domain Group Processes updates and clarifications (motion)
- Forum finance and incorporation discussion
- Actions and next meetings

**Comment by funksoup** [comment]
<img width="699" alt="Screen Shot 2022-11-16 at 5 48 11 PM" src="https://user-images.githubusercontent.com/34685781/202231340-d2c64194-a1d8-4297-b54a-648066bb234d.png">

I entered late, so here's what I caught:
Discussion of board seats & fiduciary responsibilities, possible fees to fund.

Suggestion by Cloudonix / Eric Klein:
As all principals agreed to the potential of having to pay we should look at nominal principal charges:
- Large company (1k plus people or revenue based) = $5k per year
- Smaller Company - $1k per year
- NGO / School = $100 or $500 per year"
700 companies are principals = a min of $700k annual budget

Responses some agree; others are critical, i.e.:
Berkeley Synthetic - Matt White to Everyone (5:48 PM):
Agreed with Stephan, there are smaller companies that will balk at adding in fees including startups that may be good contributors to the forum who could be dissuade from continuing with the Forum.

Huawei: Paul Higgs to Everyone (5:52 PM)
We need to always be financially viable, so ensuring dues cover operatonally costs is critical - right now I see the runrate nearer $250K annually. Donations can be used to seed non-operational activities

Perry Ellis / Eric Ackerman (who also is on the election ballot for co-chair of Standards Registry working group): He voiced a point about making the entry level lower -- i.e., he would rather have a lower / nominal amount per member but have many members/principals. $1000 is a lot of money in some countries, etc.

More discussion that fee tiers would be like a 'bait and switch', i.e., folks were encouraged to join MSF and told it would be free, but if fee structure gets implemented, then it's not. 

Actions and next meetings:
<img width="700" alt="Screen Shot 2022-11-16 at 6 01 21 PM" src="https://user-images.githubusercontent.com/34685781/202233301-3a58a34f-49cc-4c8b-87f8-71c0e02b5047.png">

**Comment by funksoup** [comment]
Water cooler chat:

- Daly Realism / Leonard:
Board is pay to play? So how do we ensure diversity. 
- Neil's response: Board is elected. We should do a member poll to find out what appetite level is among participants. 
- Daly Realism / Leonard: How about if those who can pay, do pay, and then others who cannot, are elected.
- Neil: Everyone needs to be elected to ensure all members are voted in equally.
- Daly Realism / Leonard: Trying to find solution for members who wish to be on board but $1000 is a lot.

**Comment by humbletim** [comment]
meeting slides: [Metaverse Standards Forum Oversight #8 EMEA 16Nov22.pdf](https://github.com/omigroup/MSF-Delegates/files/10023569/Metaverse.Standards.Forum.Oversight.8.EMEA.16Nov22.pdf)

**Comment by funksoup** [comment]
[meeting_saved_chat.txt](https://github.com/omigroup/MSF-Delegates/files/10023791/meeting_saved_chat.txt)
